axiom/nr_segment_datastore.c

```diff
@@ -12,6 +12,36 @@
 #include "util_sql.h"
 #include "util_logging.h"
 
+static bool create_instance_metric(nr_segment_t* segment,
```

Comment:
could nix the `bool` return type entirely since it doesn't seem to get used in the below function calls

Reply:
It's unused but I still think that returning the bool is important for the completeness of this function. I could see at some point adding a log message, for example, if this returns false.

Reply:
How does `bool` return value make this function more complete? This function is not checking anything (`is_...`, `check_...`), it is only causing side effects (update `segment`). If anything, it could return whether it succeeded (`NR_SUCCESS`) or failed (`NR_FAILURE`) to accomplish the task.

Reply:
changed to a status in [8460dfa](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/8460dfaa7a2ffa68a7b8f8b123ba48d2717102c6)